view/frontend/web/js/view/payment/method-renderer/omise-offsite-installment-method.js

```diff
@@ -139,7 +139,7 @@ define(
              * @return {integer}
              */
             getTotal: function () {
-                return + window.checkoutConfig.totalsData.grand_total;
+                return + parseInt(window.checkoutConfig.quoteData.grand_total);
```

Comment:
The `parseInt` is unneccesary `return + window.checkoutConfig.totalsData.grand_total;` already does the numeric conversion
<img width="207" alt="image" src="https://user-images.githubusercontent.com/1510194/80570138-7b481a00-8a24-11ea-85d3-1f175d8e27b5.png">

Reply:
@jonrandy Thanks, updated.